tests/integration/test_api/test_config/test_limit_eps/test_limit_eps.py

```diff
@@ -0,0 +1,117 @@
+import pytest
```

Comment:
We should add the copyright at the top of this file. Also, I'm not sure if it is a requirement but the same docstring generally contains the test description, components, suite, targets, daemons, etc. See [test_upload_configuration.py](https://github.com/wazuh/wazuh/blob/cdc44097c91876950185faddbff7e6902a0ee5e1/tests/integration/test_api/test_config/test_upload_configuration/test_upload_configuration.py) for example. We should check if it must be added and do it accordingly.

Reply:
Added in commit 55c3deb54b1205546430149e7a80b348833da665.